back/boxtribute_server/business_logic/box_transfer/agreement/mutations.py

```diff
@@ -19,16 +18,18 @@
 @mutation.field("createTransferAgreement")
 @convert_kwargs_to_snake_case
 def resolve_create_transfer_agreement(*_, creation_input):
-    # Enforce that the user can access at least one of the specified source bases
-    # (default: all bases of the user's organisation)
-    base_ids = creation_input.get(
-        "source_base_ids",
-        [
-            b.id
-            for b in Base.select().where(Base.organisation == g.user.organisation_id)
-        ],
-    )
+    # "Source" input arguments refer to the organisation/bases of the party that
+    # initiates the agreement.
+    try:
+        # Enforce that the user can access at least one of the bases of the initiating
+        # party (default: all bases of the user's organisation)
+        base_ids = creation_input.get(
```

Comment:
what happens if source_base_ids is empty?

Reply:
It defaults to all bases that the user is authorized for (see comment above and line below)